demo/radio-button-basic-demos.html

```diff
@@ -5,22 +5,93 @@
         display: block;
       }
     </style>
+    <h3>Basic usage</h3>
+    <vaadin-demo-snippet id="basic-usage">
+      <template preserve-content>
+        <vaadin-radio-group label="Label" theme="vertical">
+          <vaadin-radio-button checked>Option one</vaadin-radio-button>
+          <vaadin-radio-button>Option two</vaadin-radio-button>
+          <vaadin-radio-button>Option three</vaadin-radio-button>
+        </vaadin-radio-group>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Disabled and disabled item</h3>
+    <vaadin-demo-snippet id="disabled-and-disabled-item">
+      <template preserve-content>
+        <vaadin-radio-group label="Disabled" disabled theme="vertical"
+        >
+        <vaadin-radio-button checked>Option one</vaadin-radio-button>
+        <vaadin-radio-button>Option two</vaadin-radio-button>
+        <vaadin-radio-button>Option three</vaadin-radio-button>
+      </vaadin-radio-group>
 
+      <span style="margin-right: 200px;"></span>
 
-    <h3>Radio Button</h3>
-    <vaadin-demo-snippet id="radio-button-basic-demos-default-radio">
+      <vaadin-radio-group label="Team" theme="vertical">
+        <vaadin-radio-button>Option one</vaadin-radio-button>
+        <vaadin-radio-button>Option two</vaadin-radio-button>
+        <vaadin-radio-button disabled>Option three</vaadin-radio-button>
+      </vaadin-radio-group>
+    </template>
+  </vaadin-demo-snippet>
+
+    <h3>Read-only</h3>
+    <vaadin-demo-snippet id="read-only" when-defined="vaadin-radio-button">
```

Comment:
You only need `when-defined="vaadin-radio-button"` if the demo snippet contains a script. 
Otherwise, you can remove it.

Reply:
Done.